DNN Platform/Library/Services/GeneratedImage/IPCount.cs

```diff
@@ -165,32 +166,10 @@ private static string BuildFilePath(string ipAddress)
         /// method to get Client ip address
         /// </summary>
         /// <returns>IP Address of visitor</returns>
+        [Obsolete("Deprecated in 9.2.0. Use UserRequestIPAddressController.Instance.GetUserRequestIPAddress")]
         public static string GetVisitorIPAddress(HttpContextBase context)
         {
-            var visitorIPAddress = context.Request.ServerVariables["HTTP_X_FORWARDED_FOR"];
-
-            if (string.IsNullOrEmpty(visitorIPAddress))
-            {
-                visitorIPAddress = context.Request.ServerVariables["REMOTE_ADDR"];
-            }
-
-            if (string.IsNullOrEmpty(visitorIPAddress))
-            {
-                visitorIPAddress = context.Request.UserHostAddress;
-            }
-
-            if (string.IsNullOrEmpty(visitorIPAddress) || visitorIPAddress.Trim() == "::1")
-            {
-                visitorIPAddress = string.Empty;
-            }
-
-            if (string.IsNullOrEmpty(visitorIPAddress))
-            {
-                //This is for Local(LAN) Connected ID Address
-                var stringHostName = Dns.GetHostName();
```

Comment:
This logic is also not used for IP address for security concern mentioned in CONTENT-8523

Reply:
@Mohtshum please have a look at this 
```
        public static string GetIPAddress(this HttpRequestMessage request)
        {
            return GetHttpContext(request).Request.UserHostAddress;
        }
```
In DotNetNuke.Web.Api HttpRequestMessageExtensions clasas. This one should be fixed as well.

Reply:
ok, thnx